packages/docs/components/toast.md

```diff
@@ -0,0 +1,374 @@
+::: slot nimatron-all
+
+<ToastPen ref="toastBox"/>
+
+# Toast
+
+Toasts are non-disruptive messaging UIs that appear at the bottom right of the interface to provide quick, at-a-glance feedback on the outcome of an action.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <aside role="status" class="ods-toast is-sample-animation-off">
+      <span class="ods-toast--icon">
+        <svg viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" class="ods-icon" aria-hidden="true"><path fill-rule="evenodd" clip-rule="evenodd" d="M7 13C10.3137 13 13 10.3137 13 7C13 3.68629 10.3137 1 7 1C3.68629 1 1 3.68629 1 7C1 10.3137 3.68629 13 7 13ZM8 4C8 4.55228 7.55228 5 7 5C6.44772 5 6 4.55228 6 4C6 3.44772 6.44772 3 7 3C7.55228 3 8 3.44772 8 4ZM8 6V11H6V6H8Z" fill="currentColor"/></svg>
+      </span>
+      <h1 class="ods-toast--title">
+        This is an info toast
+      </h1>
+      <p class="ods-toast--body">This is the default variant. It provides the user with neutral information.</p>
+      <button class="ods-toast--dismiss" aria-label="Dismiss toast" data-micromodal-close></button>
+    </aside>
+  </div>
+
+```html
+<aside role="status" class="ods-toast is-sample-animation-off">
+  <span class="ods-toast--icon">
+    <svg viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" class="ods-icon" aria-hidden="true">...</svg>
+  </span>
+  <h1 class="ods-toast--title">
+    This is an info toast
+  </h1>
+  <p class="ods-toast--body">
+    This is the default variant. It provides the user with neutral information.
+  </p>
+  <button
+    class="ods-toast--dismiss"
+    aria-label="Dismiss toast"
+  ></button>
+</aside>
+```
+
+</figure>
+
+## Behavior
+
+Toasts may be triggered by many different kinds of events, but they are always transient (self-dismissing). With this in mind, Toasts should not include any interactive or long-form content.
```

Comment:
Why should they always be transient?

Reply:
As noted above, Toasts have a very specific use case. They're a `role="status"` component intended for transient use cases, not interactions or long-form text. While they look like desktop/mobile "Notifications", their use is more akin to actual mobile Toasts.

We have some followup tickets to allow for products to modify the timing of Toasts, but Toasts will likely remain transient unless we find that other patterns or components don't solve a non-transient messaging function.

Reply:
To force content to be explained in page. Over the years, callouts are vastly overused in many UIs. This is a way to manage this.

Reply:
Makes sense, assuming we plan to support alert/banner/callout components. In the past, it was unknown if we would support these - so keeping the Toast component flexible would be an excellent strategy to support more use cases.